source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts

```diff
@@ -11,7 +11,8 @@ export const defaultMicrosoftHosts: containers.HostContainer[] = [
 	new containers.OutlookContainer("Outlook Actionable Messages", "containers/outlook-container.css"),
 	new containers.DarkTeamsContainer("Microsoft Teams - Dark", "containers/teams-container-dark.css"),
 	new containers.LightTeamsContainer("Microsoft Teams - Light", "containers/teams-container-light.css"),
-	new containers.VivaConnectionsContainer("Viva Connections", "containers/viva-connections-container.css"),
+	new containers.VivaConnectionsContainerDark("Viva Connections - Dark", "containers/viva-connections-container-dark.css"),
```

Comment:
I would suggest name `DarkVivaConnectionsContainer` and LightVivaConnectionsContainer` to match with name format like DarkTeamsContainer